autoPlay.user.js

```diff
@@ -251,7 +258,23 @@ function disableParticles() {
 	}
 }
 
+function updateCurrentTime() {
+    cHours = cTime.getUTCHours();
```

Comment:
You want this function to return current time, don't you? If so, this isn't going to work.